### PR TITLE
Redirect to root if search query is empty

### PR DESCRIPTION
**Pull request by YouFoundAlpha**

### CODE REVIEW

File: scripts/app.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+// Redirect if the URL contains an empty search query
+const urlParams = new URLSearchParams(window.location.search);
```
Comment (oyepriyansh) [inline scripts/app.js:2]: variable `urlParams` is also declared in [another line](https://github.com/oyepriyansh/DevProfiles/blob/9d598168523bfb3e88fd75bd5460f5c620cab0df/scripts/app.js#L147) as const

Comment (oyepriyansh) [inline scripts/app.js:2]: variable `urlParams` is also declared in [another line](https://github.com/oyepriyansh/DevProfiles/blob/9d598168523bfb3e88fd75bd5460f5c620cab0df/scripts/app.js#L147) as const

**Review by oyepriyansh — CHANGES_REQUESTED** [review]

**Comment by oyepriyansh** [comment]
not working

**Comment by YouFoundAlpha** [comment]
> not working

It works on fine my side 
What's the issue?

**Pull request merged**